platform/android/detect.py

```diff
@@ -134,7 +134,7 @@ def mySpawn(sh, escape, cmd, args, env):
 		env['SHLINKFLAGS'][1] = '-shared'
 		env['SHLIBSUFFIX'] = '.so'
 	elif (os.name=="nt"):
-		gcc_path=gcc_path+"/windows-x86_64/bin" #this may be wrong
```

Comment:
Android 64-bits NDK has a LONG standing issue when compiling something for ARM NEON, it simply fail, I think while it's a cross compiler, the 64-bit version of the NDK has no technical benefit on the result binaries. BTW 64-bit NDK works on Linux without issues, this is only Windows related.